core/src/main/java/org/verapdf/metadata/fixer/utils/ValidationStatus.java

```diff
@@ -45,9 +45,6 @@
  * run different handling of document.
  *
  * @author Evgeniy Muravitskiy
- * @see org.verapdf.metadata.fixer.gf.MetadataFixerImpl#fixAndSaveDocument(java.io.OutputStream, 
```

Comment:
Please return it back

Reply:
It can't be returned because `MetadataFixerImpl#fixAndSaveDocument` a private method. You don't have access for such methods in javadoc